app/src/main/java/no/ntnu/klubbhuset/ui/userviews/club/detailed/ClubDetailedMember.java

```diff
@@ -0,0 +1,64 @@
+package no.ntnu.klubbhuset.ui.userviews.club.detailed;
```

Comment:
For consistency shouldn't every fragment-class include "fragment" at the end of the name?